### DISCUSSION TITLE
NerdTree - Move back and forth between editor

### DISCUSSION THREAD

**Discussion by jcarpenter12** [JetBrains/ideavim#489] [Q&A] [answered]
Are there any plans to implement a way to switch between nerd tree and editor using the command 'Ctrl + W + W' like you can do within VIM? Love the new vim plug integration but this is a bit of a deal breaker for me unfortunately as I'm so used to using that feature to switch between the tree and the editor

Thanks

**Comment by AlexPl292** [comment, marked as answer]
Hi, yes, it's planned, but not in the short term. Please follow this ticketL https://youtrack.jetbrains.com/issue/VIM-2245
